client/app/license/license.html

```diff
@@ -71,35 +75,37 @@
                 </service-expiration-date>
             </oui-column>
 
-            <oui-action-menu align="end" compact>
+            <oui-action-menu
+                data-align="end"
+                data-compact>
                 <oui-action-menu-item
-                    text="{{:: 'license_dashboard_table_popover_details' | translate }}"
-                    href="{{ $state.href('app.license.detail', { licenseId: getLicenseId($row) }) }}">
+                    data-text="{{:: 'license_dashboard_table_popover_details' | translate }}"
```

Comment:
According to the documentation the [`text`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu#deprecated) attribut is now deprecated. Prefer using the transclude slot available.